src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -142,24 +136,22 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     void ActionMap::_PopulateAvailableActionsWithStandardCommands(std::unordered_map<hstring, Model::ActionAndArgs>& availableActions, std::unordered_set<InternalActionID>& visitedActionIDs) const
     {
         // Update AvailableActions and visitedActionIDs with our current layer
-        for (const auto& [actionID, cmd] : _ActionMap)
+        for (const auto& [_, cmd] : _ActionMap)
         {
-            if (cmd.ActionAndArgs().Action() != ShortcutAction::Invalid)
+            // Only populate AvailableActions with actions that haven't been visited already.
+            const auto actionID = Hash(cmd.ActionAndArgs());

```

Comment:
Unfortunately we still have to use `InternalActionID` to populate the available action dropdown in the SUI